### ISSUE TITLE
When I type in console and something else is printed to the console what ever I typed gets removed. But when I press enter the command is ran

### ISSUE THREAD

**Issue by Smartjer123** [GameServerManagers/LinuxGSM#1698]
Plz fix

**Comment by Scarsz** [comment]
An exemplary example of how not to use GitHub Issues. This is a problem with the server software, not LGSM.

**Comment by MarkL4YG** [comment]
Actually that is up to the terminal you're using and has to do with how the Std. IO-Streams work.

Whatever you type is sent to the stdIN stream either instantaneously orbas soon as you hit enter (depending on you terminal/shell) and whatever the program wants to send to its console is sent to the stdOUT stream.

Those two streams ___do not interact___ with each other. So actions on one do not affect the other.

Now what you're experiencing is a behavior of you terminal/shell which displays both your input and the process output in the same log:
1. You type a few characters
2. Application sends a line (followed by a ``newline`` character) -> This is ___only visually___ appended to what you've written (Remember that the stdIN is not affected by this)
3. You type the rest and hit enter -> The application receives what you typed via stdIN and does ___not know___ that you've seen something different



tl;dr: You don't need to worry about this as it is just a visual effect.

**Comment by Smartjer123** [comment]
I am just wondering if it is possible to fix this because I cant review what I am typing.

**Comment by Scarsz** [comment]
No. Sorry but you have to live with it. Certain server softwares (such as Minecraft, I believe) don't have the problem while some do. Nothing we as developers of LGSM can do.

**Comment by Smartjer123** [comment]
Ok

**Comment by MarkL4YG** [comment]
Actually there is: Use a terminal/shell that does not write stdIN and stdOUT on the same lines. 

Or wrap another one that is capable of that around lgsm.

Or just find a way not to need the server console ^^

**Comment by Smartjer123** [comment]
Like what ? I am using ssh to connect to my server.

**Comment by MarkL4YG** [comment]
The last one is propably the way to go for you. It is also the easiest.
